.github/workflows/video-integration.yml

```diff
@@ -1,10 +1,8 @@
   name: Video Integration Test
 
   on:
-    pull_request:
-      branches:
-        - master
-        - release-1.x
+    workflow_run:
+      workflows: ["Build and Test Workflow"]
```

Comment:
Checking if this and the video-test-app-integration workflows need to wait on the workflow completion?